src/screens/ShowDetailsScreen.tsx

```diff
@@ -182,17 +231,9 @@ const ShowDetailsScreen: React.FC = () => {
                         >
                             {details.title}
                         </Typ>
-                        {details.release_date && details.release_date.length === 10 && (
-                            <Typ align='left' data-testid='details-release-date'>
-                                {formatReleaseDate(details.release_date, DateSize.LONG)}
-                            </Typ>
-                        )}
+                        {handleReleaseDates()}
                         <Typ align='left'>{details.age_rating}</Typ>
-                        <Typ align='left' variant='body2'>
-                            {details.runtime && details.runtime > 0
-                                ? details.runtime + ' minutes'
-                                : 'No runtime available'}
-                        </Typ>
+                        {handleRuntimes()}
```

Comment:
I think it might make more sense to put the run time directly below the release date.